Documentation/openshift-issues.md

```diff
@@ -0,0 +1,62 @@
+---
+title: Common Issues on OpenShift
+weight: 10620
+indent: true
+---
+
+# Common Issues on OpenShift
+
+## Monitoring Ceph Cluster on OpenShift
+
+OpenShift Console uses OpenShift Prometheus for monitoring and populating data in Storage Dashboard. Following troubleshooting guide assumes that it is allowed to monitor the Ceph Cluster.
+
+1. Check if Ceph Cluster monitoring is enabled
+
+    `$ kubectl get cephcluster rook-ceph -n rook-ceph -oyaml`
+
+    ```YAML
+    apiVersion: ceph.rook.io/v1
+    kind: CephCluster
+    metadata:
+    name: rook-ceph
+    namespace: rook-ceph
+    [...]
+    spec:
+    [...]
+    monitoring:
+        enabled: true
+        rulesNamespace: "rook-ceph"
+    [...]
+    ```
+
+    > **Note:** Follow [ceph-monitoring/prometheus-alerts](https://github.com/rook/rook/blob/master/Documentation/ceph-monitoring.md#prometheus-alerts) to enable Ceph Cluster monitoring.
+
+2. Check for label on namespace
+
+    `$ kubectl get namespace rook-ceph --show-labels | grep openshift.io/cluster-monitoring=true`
+
+    ```bash
+    rook-ceph   Active   15h   openshift.io/cluster-monitoring=true
+    ```
+
+    If not present, label the namespace with,
+
+    `$ kubectl label namespace rook-ceph "openshift.io/cluster-monitoring=true"`
+
+3. Check if service monitor is present
```

Comment:
The service monitor should exist after following the instructions with the doc in step 1, right? Seems like this step is already covered by the other doc.

Reply:
We haven't mentioned anything as such in the other guide. It just talks about how to enable and everything will be done behind the scene.

Will move this to the Troubleshooting section.